src/tests/integration-test

```diff
@@ -574,6 +575,7 @@ class Manager(UDisksTestCase):
             self.assertNotEqual(loop, None)
             self.assertEqual(obj.get_property('filesystem'), None)
 
+            time.sleep(1) # time to settle the backing file
```

Comment:
Do we need these two ``sleep()`` calls here even with ``bd_loop_get_autoclear()`` fixed?